Ease of use changes

MllwAlms opened the pull request:
- URL connection success notice
- Replace input with argv
- Automates license header update using uncurl and URL

pythonkenyard commented:
You should allow both options for situation where users dont know to add it as sys arg. 

try:
    MDP_URL = sys.argv[1]
    lic_url = sys.argv[2]
except:
    MDP_URL = input('\nInput MPD URL: ')
    lic_url = input('License URL: ')

MllwAlms commented:
Added

> You should allow both options for situation where users dont know to add it as sys arg.
> 
> try: MDP_URL = sys.argv[1] lic_url = sys.argv[2] except: MDP_URL = input('\nInput MPD URL: ') lic_url = input('License URL: ')
> 
> edit: comments dont allow indent formatting it seems

Added
